components/forms/form-builder/form-sections/ImportantText.tsx

```diff
@@ -93,24 +93,30 @@ const unresolvedDecsText =
   "If you know of any UNRESOLVED Significant Events, Complaints, Other investigations since your last ARCP/RITA/Appraisal, please provide a brief summary, including where you were working, the date of the event, and your reflection where appropriate. If known, please identify what investigations are pending relating to the event and which organisation is undertaking the investigation.";
 
 const ltftDiscussionText1 =
-  "Please provide the contact details of the Training Programme Director (TPD) you have discussed your Changing hours (LTFT) proposals with prior to completing this form. ";
+  "Please provide the contact details of the Training Programme Director (TPD) you have discussed your Changing hours (LTFT) proposal with prior to completing this form. ";
 
 const ltftDiscussionText2 =
-  "When you submit this appication, your TPD will receive an email which will include a summary of your proposals.";
+  "When you submit this application, your TPD will receive an email which will include a summary of your proposal.";
 
 const ltftOtherDiscussionsText =
-  "If applicable, please provide details of any other discussions you have had concerning your Changing hours (LTFT) proposals.";
+  "If applicable, please provide details of any other discussions you have had concerning your Changing hours (LTFT) proposal.";
 
-const ltftReasonsText1 =
-  "Please provide the reason for your Changing hours (LTFT) request. If you have more than one reason, please select multiple reasons.";
+  const ltftDiscussionText3 =
+    "For information on Professional support contact your <a href='/support' target='_new'>local office<a/>.";
+
+  const ltftReasonsText1 =
+    "Note: the reason for applying is for reporting purposes only and has no bearing on the decision-making process.";
 
 const ltftReasonsText2 =
+  "Please provide the reason for your Changing hours (LTFT) request. If you have more than one reason, please select multiple reasons.";
+
+const ltftReasonsText3 =
   "If your reason isn't in the list then please select 'other reason' and give details in the space provided.";
 
 const generateTextElement = (texts: string[]) => (
   <span>
-    {texts.map(text => (
-      <p key={text}>{text}</p>
+    {texts.map((text, index) => (
+      <p key={index} dangerouslySetInnerHTML={{ __html: text }} />
```

Comment:
Its not great, but if we are managing the text content this should be ok?

Reply:
We've done it in other places that we control the content, so should be fine